src/Parser/Druid/Balance/CombatLogParser.js

```diff
@@ -32,13 +34,15 @@ class CombatLogParser extends MainCombatLogParser {
     abilities: Abilities,
     lsempowerment: LEmpowerment,
     swempowerment: SEmpowerment,
+    cooldownThroughputTracker: CooldownThroughputTracker,
     //Spells
     fullmoon: FullMoon,
     halfmoon: HalfMoon,
     moonfire: Moonfire,
     newmoon: NewMoon,
     sunfire: Sunfire,
     unempoweredLS: UnempoweredLs,
+    //moonSpells: MoonSpells,
```

Comment:
If you are removing the module for good, delete it instead of commenting out. If it's only temporary, it's fine.